CorsixTH/Lua/dialogs/message.lua

```diff
@@ -105,6 +105,11 @@ function UIMessage:adjustToggle()
 end
 
 function UIMessage:openMessage()
```

Comment:
As a rule of the thumb, please document the function when you touch it.

You have spend time to understand what it does, don't make others spend that same effort again, it's a waste of time.